src/WellIO.jl

```diff
@@ -1,5 +1,20 @@
 module WellIO
 
-# Write your package code here.
+using FileIO
+using Tables
 
+function load(filepath::String)
+    file = FileIO.query(filepath)
+    data = open(file) do f
+           read(f, String)
+        end
+    version_block = match(r"~V(.*?)~"s, data)
+    wellinfo_block = match(r"~W(.*?)~"s, data)
+    curveinfo_block = match(r"~C(.*?)~"s, data)
+    otherinfo_block = match(r"~O(.*?)~"s, data)
+    datatable_block = match(r"~A(.*?)\Z"s, data)
+    return(version_block, wellinfo_block, curveinfo_block, otherinfo_block, datatable_block)
```

Comment:
Please avoid the `return` keyword. It is only used when we want to return prematurely from a function in idiomatic Julia. Also, any function that returns more than 3 arguments indicates a flaw in design. We should have a `struct Metadata ... end` type to hold these sections of the header. A function would take the header string and would convert it to a `Metadata` instance for example. Makes sense?